src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -300,6 +300,7 @@ public void TestConditionalExpressions()
         {
             VerifySyntax<BinaryExpressionSyntax>(_g.CoalesceExpression(_g.IdentifierName("x"), _g.IdentifierName("y")), "(x) ?? (y)");
             VerifySyntax<ConditionalExpressionSyntax>(_g.ConditionalExpression(_g.IdentifierName("x"), _g.IdentifierName("y"), _g.IdentifierName("z")), "(x) ? (y) : (z)");
+            VerifySyntax<ConditionalAccessExpressionSyntax>(_g.ConditionalAccessExpression(_g.IdentifierName("x"), _g.IdentifierName("y")), "(x)?.y");
```

Comment:
Let's add some more tests as well.  We'd want this to work for things like   a?.b.c  etc.